lte/gateway/python/scripts/config_stateless_agw.py

```diff
@@ -125,11 +138,7 @@ def enable_stateless_agw():
         sys.exit(return_codes.STATELESS.value)
     for service, config, value in STATELESS_SERVICE_CONFIGS:
         cfg = load_override_config(service) or {}
-        if service == "pipelined":
-            cfg[config] = False
-        else:
-            cfg[config] = True
-
+        cfg[config] = value
```

Comment:
I believe this is done because `clean_restart` is on the inverse of all the other variables, we might need a better check than checking the service itself then